mapadroid/worker/WorkerMITM.py

```diff
@@ -191,7 +191,16 @@ def _check_for_data_content(self, latest_data, proto_to_wait_for: ProtoIdentifie
         if latest_proto_data is None:
             return LatestReceivedType.UNDEFINED, data_found
         latest_proto = latest_proto_data.get("payload")
-        key_to_check: str = "wild_pokemon" if mode in ["mon_mitm", "iv_mitm"] else "forts"
+
+        if mode in ["mon_mitm", "iv_mitm"]:
```

Comment:
Could use enums like in WorkerFactory https://github.com/Map-A-Droid/MAD/blob/90e5f5b52f546b36f48ef18446125afa298c6c17/mapadroid/worker/WorkerFactory.py#L184

Reply:
Would prefer to keep this out. The `mode` variable is only used in this line, so converting it first to an enum seems like unnecessary load to me.

https://github.com/Map-A-Droid/MAD/blob/master/mapadroid/utils/MappingManager.py#L247-L249

it gets that variable from this function, which is referenced a lot by other modules that are also using strings instead of enums. Is there something I'm missing?

Reply:
WorkerType.IV_MITM.value evaluates to "iv_mitm" ;)